src/app/game/memory-flip-card/page.tsx

```
@@ -0,0 +1,351 @@
+      <div className={`${styles.gameBoard} ${styles[`grid${gridCols}`]}`}>
+        {cards.map((card) => (
+          <div
+            key={card.id}
+            className={`${styles.card} ${
+              card.isFlipped || card.isMatched ? styles.flipped : ''
+            } ${card.isMatched ? styles.matched : ''}`}
+            onClick={() => handleCardClick(card)}
+          >
+            <div className={styles.cardInner}>
+              <div className={styles.cardFront}>
+                <span className={styles.cardSymbol}>?</span>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)